app/routes.py

```diff
@@ -1,9 +1,34 @@
 # View functions
-from flask import render_template, url_for
-from app import app
+from flask import render_template, url_for, flash, redirect
+from app.forms import RegistrationForm
+from flask_login import current_user
+from app.models import User
+from app import app, db
 
 
 @app.route('/')
 def index():
     feed_blank = 'Podcast Main page: RSS feed'
     return render_template('index.html', feed_blank=feed_blank)
+
+
+@app.route('/')
+def login():
```

Comment:
second '/' route?

Reply:
This if only for tests. Sorry about tha.